cmd/ipfs/daemon.go

```diff
@@ -267,11 +267,6 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 			return
 		}
 
-		repo, err = fsrepo.Open(cctx.ConfigRoot)
```

Comment:
I'm pretty sure this is still necessary.

Reply:
This will happen in https://github.com/ipfs/go-ipfs/pull/4948/files#diff-153cb71153ee178fe2869148600ce8d5R113